### ISSUE TITLE
Search functionality

### ISSUE THREAD

**Issue by GoogleCodeExporter** [heron-mc/heron-mc#7]
```
Search functionality needed to search for features within the active (wfs) 
layer/theme. The search function is theme aware and creates different WFS 
filter depending on the active theme.
```

Original issue reported on code.google.com by `ott...@gmail.com` on 22 Jun 2010 at 10:10
- Blocked on: #6
- Merged into: #30
